tests/integration-tests/tests/update/test_update.py

```diff
@@ -591,30 +613,40 @@ def _test_update_queue_strategy_with_running_job(
         global_custom_ami=pcluster_ami_id,
         custom_ami=pcluster_copy_ami_id,
         updated_compute_root_volume_size=updated_compute_root_volume_size,
+        queue_update_strategy=queue_update_strategy,
     )
     cluster.update(str(updated_config_file))
     # check chef client log contains expected log
     assert_errors_in_logs(
         remote_command_executor,
         ["/var/log/chef-client.log"],
-        ["Queue update strategy is \\(DRAIN\\)", "Adding queue \\(queue2\\) to list of queue to be updated"],
+        [
+            f"Queue update strategy is \\({queue_update_strategy}\\)",
+            "Adding queue \\(queue2\\) to list of queue to be updated",
+        ],
     )
 
-    # after cluster update, check if jobs are still in running state
+    # after cluster update, check if queue1 node state are in working state
+    ec2 = boto3.client("ec2", region)
     scheduler_commands.assert_job_state(queue1_job_id, "RUNNING")
-    scheduler_commands.assert_job_state(queue2_job_id, "RUNNING")
-    # assert queue1 node state are in working state
     queue1_nodes = scheduler_commands.get_compute_nodes("queue1")
     assert_compute_node_states(scheduler_commands, queue1_nodes, expected_states=["mixed", "allocated"])
-    # assert queue2 node state are in draining status
+    # check queue1 AMIs are not replaced
+    _check_queue_ami(cluster, ec2, pcluster_ami_id, "queue1")
+
     queue2_nodes = scheduler_commands.get_compute_nodes("queue2")
-    assert_compute_node_states(scheduler_commands, queue2_nodes, expected_states=["draining", "draining!"])
-    # retrieve queue2 instance again, check ami, they are not replaced
     ec2 = boto3.client("ec2", region)
-    _check_queue_ami(cluster, ec2, pcluster_ami_id, "queue1")
-    _check_queue_ami(cluster, ec2, pcluster_ami_id, "queue2")
-    # requeue job in queue2 to launch new instances for nodes
-    remote_command_executor.run_remote_command(f"scontrol requeue {queue2_job_id}")
+    # assert queue2 node state are in expected status corresponding to the queue strategy
+    if queue_update_strategy == "DRAIN":
+        scheduler_commands.assert_job_state(queue2_job_id, "RUNNING")
+        _check_queue_ami(cluster, ec2, pcluster_ami_id, "queue2")
+        assert_compute_node_states(scheduler_commands, queue2_nodes, expected_states=["draining", "draining!"])
+        # requeue job in queue2 to launch new instances for nodes
+        remote_command_executor.run_remote_command(f"scontrol requeue {queue2_job_id}")
+    elif queue_update_strategy == "DOWN":
+        scheduler_commands.assert_job_state(queue2_job_id, "PENDING")
```

Comment:
can you please confirm that jobs go directly into PENDING?

Reply:
Yes, after POWER_DOWN_FORCE on running job, the job turn from R to PD state